eclass/bitcoincore.eclass

```diff
@@ -159,7 +162,7 @@ bitcoincore_common_depend_use() {
 	in_bcc_iuse "$1" || return
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $1? ( $2 )"
 }
-bitcoincore_common_depend_use upnp net-libs/miniupnpc
+bitcoincore_common_depend_use upnp '>net-libs/miniupnpc-1.9.20150915'
```

Comment:
Literally *anything* else is/should be desirable over leaving users vulnerable to this. "Breaking" the depgraph here gives the user the option between knowingly keywording a safe version of miniupnpc, or disabling USE=upnp to build without it.